corehq/apps/sms/tasks.py

```diff
@@ -604,3 +610,25 @@ def queued_sms():
 
 
 datadog_gauge_task('commcare.sms.queued', queued_sms, run_every=crontab())
+
+if settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS:
```

Comment:
Shouldn't this go somewhere under `custom/icds`?

Reply:
+1

Reply:
https://github.com/dimagi/commcare-hq/pull/25882/commits/8a9620643e2cbf28e4c9d514ed0c933f05e6ad9d